run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script is to be run from inside the Sonobuoy pod
+
+curl -sL https://raw.githubusercontent.com/linkerd/website/master/run.linkerd.io/public/install | sh
+
+/bin/bash -c "./conformance -integration-tests -linkerd /root/.linkerd2/bin/linkerd -ginkgo.v -linkerd-namespace l5d-conformance --ginkgo.reportFile=/tmp/results/report.xml"
+
+
+tar czvf /tmp/results/results.tar.gz -C /tmp/results .
+
+echo -n /tmp/results/results.tar.gz > /tmp/results/done
```

Comment:
I've introduced this script after going through the Sonobuoy docs. Instead of cluttering the plugin file with a lot of commands, the container simply uses this as a start up script to install the specified version of linkerd2 binary (which is read from the `LINKERD2_VERSION` env variable), and initiate the tests.